packages/amplify-cli/src/__tests__/commands/status.test.ts

```diff
@@ -1,14 +1,12 @@
 import { stateManager, pathManager } from '@aws-amplify/amplify-cli-core';
-import { readProjectSchema } from 'graphql-transformer-core';
+import { showApiAuthAcm } from '@aws-amplify/amplify-category-api';
```

Comment:
what's `Acm` ? public apis should avoid acronyms of possible.
(for the record - it seems this has shipped already)